### ISSUE TITLE
meeting at the blackwing coven

### ISSUE THREAD

**Issue by LightBringe1972** [WoWManiaUK/Blackwing-Lair#4938]
**Links:**

from WoWHead or our Armory

**What is happening:**
the mobs are not dropping the costume scraps

**What should happen:**
after the mobs drop the costume scraps, you disguise yourself in order to listen in onthe meeting.
**Is it crashing the server?:**
no
**Other Information:**

**Comment by Echinaceax** [comment]
@mfranca0009 
could be due to this fix, the item was dropping without having the quest #748

**Comment by mfranca0009** [comment]
@LightBringe1972 Please provide links for the quest and relevant information that is related to the quest under the **Links** section next time.

**Comment by mfranca0009** [comment]
@Echinaceax  Yes, that fix actually prevented them from dropping at all, because they aren't a specific item drop for a quest. These items are suppose to be used to create a disguise not for a turn-in requirement.

I have corrected the loot tables of the NPCs that are suppose to drop them, because they were incorrect.
I have corrected the conditions to when these items should be dropping.

Should be all set and needs testing in PTR 👍
